force-app/main/default/classes/CheckFulfillmentOrders_Batch.cls

```diff
@@ -0,0 +1,39 @@
+public class CheckFulfillmentOrders_Batch Implements Database.Batchable<sObject>, Database.Stateful {
+
+    public Database.QueryLocator start(Database.BatchableContext bc) {
+        FO_immediate_attention_delay__mdt fOImmediateAttentionDelayMdt = 
+            [SELECT Number_Of_Hours__c FROM FO_immediate_attention_delay__mdt];
+
+        datetime searchDate = datetime.now().addHours(-fOImmediateAttentionDelayMdt.Number_Of_Hours__c.intValue());
+
+        List<Order> orders = [SELECT Id FROM Order WHERE OrderedDate < :searchDate];
+
+        String ids = '';
```

Comment:
Since you will remove the List<Orders> this variable and the for below won't be needed anymore.